terraform/aws/analytical-platform-management-production/terraform-state/data.tf

```diff
@@ -7,3 +7,38 @@ data "aws_iam_session_context" "session" {
 
   arn = data.aws_caller_identity.session.arn
 }
+
+data "aws_region" "current" {}
+
+data "aws_iam_roles" "analytical_platform_team_access_role" {
```

Comment:
I understand why limiting access would be a good idea, but I think we might be called upon (maybe only initially) to help so worthwhile to add adminaccess roles in data-eng-prod unless you think it's better we're able to plan from management?

Reply:
And also data-prod and so on.

Reply:
I _thought_ I'd made it so **our** permissions and local workflow remain the same; as in

1. exec into management prod
1. init
1. ???
1. profit

DEs will assume the relevant DE role for that account